<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MqsEncoder</class>
 <widget class="QWidget" name="MqsEncoder">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encoder Properties</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/MediaQuickSync/resources/GoContactSyncLogo.ico</normaloff>:/MediaQuickSync/resources/GoContactSyncLogo.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,6,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Video Encoding</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>76</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Quality:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_preset">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>45</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ultrafast</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>superfast</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>veryfast</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>faster</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fast</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>slow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>slower</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>veryslow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>placebo</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="combo_quality">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>75</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>47</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Preset:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_encoder">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>105</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>105</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Encoder:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_buffer">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>45</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>512</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1024</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2048</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4096</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8192</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12288</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16384</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24576</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32768</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="combo_bitrate">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>75</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>384</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>512</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>768</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1024</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1536</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2048</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2560</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3072</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4096</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6144</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8192</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>47</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Buffer:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>76</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Bitrate:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_name">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>18</y>
        <width>347</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Audio Encoding</string>
     </property>
     <widget class="QComboBox" name="combo_audio_format">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>57</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>44100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22050</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Format:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Bitrate:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_audio_bitrate">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>55</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>56</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>80</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>96</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>112</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>160</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>224</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>256</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>320</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>26</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Encoder:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_audio_encoder">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>25</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>mp3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>aac -strict 2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>26</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Source:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_audio_source">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>24</y>
        <width>135</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
